velox/docs/develop/aggregate-functions.rst

```diff
@@ -169,6 +187,15 @@ function's argument type(s) wrapped in a Row<> even if the function only takes
 one argument. This is needed for the SimpleAggregateAdapter to parse input
 types for arbitrary aggregation functions properly.
 
+A FunctionState struct needs to be declared in the simple aggregation function
+class, it is used to hold the function-level variables that are typically
```

Comment:
nit: ...in the simple aggregation function class. FunctionState is initialized once when the aggregation function is created and used at every row when adding inputs to accumulators or extracting values from accumulators...